### PR TITLE
Test coverage

### PR DESCRIPTION
**Pull request by Yaqf**

### CODE REVIEW

**Comment by Yaqf** [comment]
https://github.com/FlowerLab/sorbifolia/issues/46

File: strutils/utils_test.go [hunk 1]
```diff
@@ -20,27 +19,29 @@ var checkUTF8LenTests = []struct {
...
+	{[]byte{0xf1, 0b10110001, 0b10011000}, 1},
 }
 
 func Test_checkUTF8Len(t *testing.T) {
 	t.Parallel()
 
 	for _, v := range checkUTF8LenTests {
-		_, size := utf8.DecodeRune(v.b)
-		if checkUTF8Len(v.b) != size {
+		// _, size := utf8.DecodeRune(v.b)
```
Comment (Aoang) [inline strutils/utils_test.go:43]: remove it

File: strutils/utils_test.go [hunk 2]
```diff
@@ -20,27 +19,29 @@ var checkUTF8LenTests = []struct {
...
-	{[]byte{0xa9}, 4}, // xx
+	{[]byte{0xa9}, 1}, // xx
 
-	{[]byte{}, 1}, // line 6
+	{[]byte{}, 0}, // line 6
 
 	// add test for p[1] < accept.lo || accept.hi < p[1]
-	{[]byte{0xc2, 0b00110001}, 2}, // line 22
-	{[]byte{0xc2, 0b11111111}, 2}, // line 22
+	{[]byte{0xc2, 0b00110001}, 1}, // line 22
```
Comment (Aoang) [inline strutils/utils_test.go:27]: Why there are two tests for a case

**Review by Aoang — CHANGES_REQUESTED** [review]

File: strutils/utils_test.go (hunk 2 above)
Comment (Yaqf) [reply strutils/utils_test.go:27]: one for` p[1] < accept.lo` and another for `accept.hi < p[1]`

File: strutils/reverse_test.go [hunk 3]
```diff
@@ -29,6 +29,24 @@ func TestReverseBytes(t *testing.T) {
 	}
 }
 
+func TestCoverage(t *testing.T) {
```
Comment (Aoang) [inline strutils/reverse_test.go:32]: Can you solve this problem like Test_checkUTF8Len

**Review by Aoang — CHANGES_REQUESTED** [review]

File: strutils/reverse_test.go (hunk 3 above)
Comment (Yaqf) [reply strutils/reverse_test.go:32]: ok，i'll solve, how about like this `Test_ReverseBytesEmpty`

Comment (Aoang) [reply strutils/reverse_test.go]: I mean coverage is done by looping test cases, not by individual test methods

Comment (Yaqf) [reply strutils/reverse_test.go]: sorry !  is my fault

**Review by Aoang — APPROVED** [review]

**Pull request merged**